### ISSUE TITLE
Rename brl:dotNumber to brl:cellType

### ISSUE THREAD

**Issue by egli** [daisy/ebraille#194]
Makes the intention behind the field more clear

**Comment by mattgarrish** [comment]
Closing this issue as the name change was included in #209

**Comment by mattgarrish** [comment]
Reopening this because of the proposed change of prefix from brl: to a11y: in #216 

The [question](https://github.com/daisy/ebraille/pull/216#pullrequestreview-2156979917) is whether the property still makes sense without a braille sounding prefix to it.

**Comment by wfree-aph** [comment]
@mattgarrish It seems like it doesn't really make sense without the braille association in the prefix. Could it be changed to a11y:braillecelltype?

**Comment by mattgarrish** [comment]
Yes. The only thing I'd suggest differently is to keep the camel case naming we use for the other properties, so make it a11y:brailleCellType
